docs/ReDoc_Research.md

```diff
@@ -1,15 +1,26 @@
 # ReDoc Research
 
 ## Overview
-Give a brief overview.
+This is an analysis of the ReDoc API visualizer that allows for projects to publish their API as an interactive documentation. It will act as an inspiration when building upon the ONTAP REST API.
 
 ## Available Features
-Disclose key available features here.
+- Server Side Rendering
+- Interactive documentation for nested objects
+- Deep Search
+- Code samples for each request
 
 ### Features Missing Currently In Swagger
-What features does Swagger have that this UI does not? Any extra features in this UI that Swagger doesn't have.
+- Interactive documentation for nested objects
+- Deep Search
 
 ## Pros / Cons
 ### Pros:
+- The code to execute for a particular request is already there without needing to be manually selected.
+- You don’t need to manually execute the code to get a particular result; it already has the output for some generic parameters.
+- The description of the request is clear along the left-hand side.
+- It has a deep-search feature that returns all of the requests with the keyword.
 
 ### Cons:
+- The different types of requests aren’t grouped together within each endpoint.
+- For a fair amount of requests, not all of the possible results of the execution of those requests are shown. For example, some requests only show the 200, and others only show the 400 or 405.
```

Comment:
@ColemanKobe Thanks for doing the background research on ReDoc. I just had a question regarding this part of the `Cons` section. I've noticed that other API visualizers also do not show all the response codes. So is it an issue with the visualizer itself or it has something to do with the API given and certain methods(get, put, post, etc.)?

Reply:
I think it would have something to do with the API itself, but I think that the visualizer should at least give some pseudo-code that works and some that doesn't, giving the various errors that could happen.